src/vs/base/browser/ui/selectBox/selectBox.ts

```diff
@@ -115,31 +333,87 @@ export class SelectBox extends Widget {
 		dom.addClass(container, 'select-container');
 		container.appendChild(this.selectElement);
 		this.setOptions(this.options, this.selected);
-
 		this.applyStyles();
 	}
 
 	public style(styles: ISelectBoxStyles): void {
-		this.selectBackground = styles.selectBackground;
-		this.selectForeground = styles.selectForeground;
-		this.selectBorder = styles.selectBorder;
+		const content: string[] = [];
+
+		this.styles = styles;
+
+		// We can only style non-native select mode
+		if (!this._useNativeSelect) {
+
+			if (this.styles.selectOptionCheckedBackground) {
+				content.push(`.monaco-shell .select-dropdown-container .selectbox-dropdown-list-container .monaco-list .monaco-list-row.focused:not(:hover) { background-color: ${this.styles.selectOptionCheckedBackground} !important; }`);
```

Comment:
@cleidigh again please express the DOM hierarchy if possible, e.g. `.select-dropdown-container .selectbox-dropdown-list-container` => `.select-dropdown-container > .selectbox-dropdown-list-container`

Reply:
Done.